### PR TITLE
add Dlang hunt web framework

### PR DESCRIPTION
**Pull request by viile**

### CODE REVIEW

**Comment by nbrady-techempower** [comment]
@viile Thanks for the contribution! A few things to start. Please add:
`    - "TESTDIR=D/hunt"` 
to the `.travis.yml` file in the project root. Also, please include a `README.md` similar to [this one](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Perl/mojolicious/README.md) so we can get some more information about the framework.

Also the `application.conf` looks like it just has generic connection urls for the databases. Make sure to add the proper credentials.

**Comment by viile** [comment]
@nbrady-techempower  has been added

File: frameworks/D/hunt/benchmark_config.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "framework": "hunt",
+  "tests": [{
+    "default": {
+      "setup_file": "setup-mysql",
```
Comment (tchaloupka) [inline frameworks/D/hunt/benchmark_config.json]: There is only setup.sh provided with the PR, not setup-mysql.sh

File: frameworks/D/hunt/setup.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Clean any files from last run
```
Comment (tchaloupka) [inline frameworks/D/hunt/setup.sh]: There should be something like: `fw_depends mysql dlang` for this to work I guess.

See: https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/D/vibed/setup.sh#L3

File: frameworks/D/hunt/benchmark_config.json (hunk 1 above)
Comment (viile) [inline frameworks/D/hunt/benchmark_config.json]: oh,it's my fault

File: frameworks/D/hunt/setup.sh (hunk 2 above)
Comment (viile) [inline frameworks/D/hunt/setup.sh]: this is not necessary

Comment (nbrady-techempower) [inline frameworks/D/hunt/setup.sh]: @viile the `travis.yml` file you need to add to is in the project root, not the framework root. This is what runs all the tests. Also, don't add any languages to that `travis.yml` file. You will need to setup your dependencies as @tchaloupka suggested. Travis is meant to verify the framework works in our environments.

File: frameworks/D/hunt/setup.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+#fw_depends mongodb dlang
```
Comment (nbrady-techempower) [inline frameworks/D/hunt/setup.sh]: you're probably going to need to uncomment this

File: frameworks/D/hunt/.travis.yml [hunk 4]
```diff
@@ -0,0 +1 @@
+language : d
```
Comment (nbrady-techempower) [inline frameworks/D/hunt/.travis.yml]: This file isn't needed

**Comment by nbrady-techempower** [comment]
@tchaloupka Looks like this is passing now, thanks! We just finished the last preview round for round 14. As soon as Round 14 is done, we will merge this in!

**Pull request merged**